development/src/abilities/Abolished.js

```diff
@@ -105,11 +107,12 @@ G.abilities[7] =[
 	query : function() {
 		var ability = this;
 		var crea = this.creature;
+		var range = this.timesUsed + 7
```

Comment:
The range was supposed to start increasing by 1 only when the ability reaches the upgrade threshold. So the first time using an upgraded Wild Wire, range would always be 8, regardless the number of uses required for an ability to upgrade. Will test soon and also consider semantics regarding upgrade description.